ament_cmake_gtest/cmake/ament_add_gtest.cmake

```diff
@@ -25,6 +25,8 @@
 # :type target: string
 # :param ARGN: the list of source files
 # :type ARGN: list of strings
+# :param RUNNER: the path to a non default test runner to use
```

Comment:
Since the user could pass the default test runner explicitly please update the line to:

> :param RUNNER: the path to the test runner script (default: see `ament_add_test`).

Same below.